environments/testing/secret.yaml

```diff
@@ -0,0 +1,16 @@
+# type: s3
+# config:
+#   bucket: thanos
+#   endpoint: minio.minio.svc.cluster.local:9000
+#   insecure: true
+#   access_key: minio
+#   secret_key: minio123
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: thanos-objectstorage
+  namespace: observatorium
+type: Opaque
+data:
```

Comment:
Any reason why not to make use of [stringData](https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret-manually) here?

Reply:
yeah +1 no need to have this bas64 encoded in this test file

Reply:
or ever for that fact if it's really just text data :man_shrugging:

Reply:
Will update that one now, yep :+1: